project-metrics/metrics_service/templates/show_metrics.html

```diff
@@ -0,0 +1,122 @@
+{% from 'metric_guage.html' import metric_guage %}
+
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Project Metrics: {{ github_repo }}</title>
+
+  <style type="text/css">
```

Comment:
Yes, this is ugly. But serving static files in flask is a whole thing so, until we know what we're actually doing with this page, embedding it here seems reasonable.

Reply:
This is a standard for AMP pages. In fact, have you considered making this page an AMP page? ;)

Reply:
Shhhh...

Reply:
I can do that in a later PR if you think it's necessary :stuck_out_tongue: